src/sentry/static/sentry/app/constants/index.tsx

```diff
@@ -255,3 +255,6 @@ export const IS_CI = !!process.env.IS_CI;
 export const NODE_ENV = process.env.NODE_ENV;
 export const DISABLE_RR_WEB = !!process.env.DISABLE_RR_WEB;
 export const SPA_DSN = process.env.SPA_DSN;
+
+// If an organization has global views, this constant refers to an all project selection.
+export const ALL_PROJECTS = -1;
```

Comment:
This constant already exists in app/constants/globalSelection as ALL_ACCESS_PROJECTS

Reply:
Ah nice. https://github.com/getsentry/sentry/pull/20161/commits/e40f4c6979ab0a5c87c192e73babc5507f979915